webapp/locales/en.json

```diff
@@ -15,7 +15,8 @@
     "followers": "Followers",
     "following": "Following",
     "shouted": "Shouted",
-    "commented": "Commented"
+    "commented": "Commented",
```

Comment:
Oh yes, providing translations right from the beginning :sparkling_heart:

Reply:
:smile_cat: